### PR TITLE
fix: Add postbuild script to web package to fix react build

### PR DESCRIPTION
**Pull request by ethanWallace**
# Summary | Résumé

Add a postbuild script that adds `// @ts-expect-error` to two files generated for the react package build. This will allow the react package to build.

**Note**: Targeting the `main` branch for the tests to run

### CODE REVIEW

File: packages/react/scripts/buildPatch.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  {
+    path: path.join(
+      __dirname,
+      '../src/components/stencil-generated/react-component-lib/utils/index.tsx',
+    ),
+    target: 'return React.forwardRef(forwardRef);',
+    replace: '// @ts-expect-error\n  return React.forwardRef(forwardRef);',
+  },
+  {
+    path: path.join(
```
Comment (daine) [inline packages/react/scripts/buildPatch.js:14]:
```suggestion
    filePath: path.join(
```

File: packages/react/scripts/buildPatch.js
Comment (daine) [inline packages/react/scripts/buildPatch.js:1]: Could we also change this to another name? `buildPatch` implies we're building a patch, but we're actually applying patches here within a pre-build script. Will `patchFiles.js` work? Or `patchScript.js`?

File: packages/react/scripts/buildPatch.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    // Write the updated content back to the file
+    fs.writeFile(file.path, result, 'utf8', err => {
+      if (err) {
+        console.error('Error writing to the file:', err);
+        return;
+      }
+    });
+  });
+});
```
Comment (daine) [inline packages/react/scripts/buildPatch.js:43]: Can we avoid nested callbacks to keep it simpler and avoid callback loops?
Can we also try and make it so it asynchronously just to make it more efficient? We can use either promises or async/await with `.map`?

**Review by daine — CHANGES_REQUESTED** [review]
Initial review - found a few things we can improve a little on, I'll still continue testing

**Comment by ethanWallace** [comment]
Closing this pull request, work will continue in https://github.com/cds-snc/gcds-components/pull/709

**Pull request closed without merging**